docs/src/pages/components/data-grid/columns/columns.md

```diff
@@ -84,7 +84,10 @@ Alternatively, to disable all columns resize, set the prop `disableColumnResize=
 
 {{"demo": "pages/components/data-grid/columns/ColumnSizingGrid.js", "disableAd": true, "bg": "inline"}}
 
-To capture changes of a column width during resizing use `onColumnResize` or `onColumnResizeCommitted`.
+To capture changes in the width of a column there are two callbacks that are called:
+
+- `onColumnResize`: Called while a column is being resized.
+- `onColumnWidthChange`: Called after the width of a column is changed, but not during resizing.
```

Comment:
The "but not during resizing" feels off. As a developer, I think that I would expect it to be always a one-on-one binding. Great to have it documented nevertheless.